src/spring-cloud/setup.py

```diff
@@ -16,8 +16,8 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-# TODO(jiec): remove this version number after verification.
-VERSION = '3.0.1-use-0301-api'
+# TODO(jiec): Fix the version number before submit PR.
+VERSION = '3.1.1-mason-app-msi'
```

Comment:
The version here is only a tmp version. Will update this before create PR to public cli extension repo.